app/views/content/blog/returning-to-teaching-with-interational-experience.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Returning to teaching with international experience
+date: "2019-10-07"
+author: Katie Lockett
+images:
+  q-katie:
+    path: "media/images/content/blog/stories-katie.jpg"
+    thumbnail_path: "media/images/content/blog/stories-katie.jpg"
+description: |-
+  Head of faculty Katie Lockett shares her story of returning to teach in England from China.
+keywords:
+  - life as a teacher
+  - day in the life
+  - classroom
+tags:
+  - non-UK teachers
```

Comment:
I think we should add another tag as well @EllieNodder 
* returning to teaching